### PR TITLE
[Release] v0.11.5b

### PR DESCRIPTION
**Pull request by ZenBre4ker**
Bump version number and add changelog.
Also replace all ' ' with <code></code>.

Waiting for #894 to be merged, which this branch is based on.

### CODE REVIEW

File: gamemodes/terrortown/gamemode/client/cl_changes.lua [hunk 1]
```diff
@@ -1230,6 +1230,61 @@ function CreateChanges()
...
+			<li>Optimized damage indicator vgui images to be smaller</li>
+		</ul>
+
+		<h2>Fixed</h2>
+		<ul>
+			<li>Fixed addon compatibility checker fussing over disabled addons</li>
+			<li>Fixed ammo entities blocking <code>+use</code> traces</li>
+			<li>Fixed double call of <code>GM:TTT2UpdateTeam</code>, when a role change leads to a team change</li>
+		</ul>
+	]], os.time({ year = 2022, month = 08, day = 05 }))
```
Comment (Histalek) [inline gamemodes/terrortown/gamemode/client/cl_changes.lua:1286]:
```suggestion
	]], os.time({ year = 2022, month = 08, day = 21 }))
```

**Review by Histalek — APPROVED** [review]

**Review by TimGoll — APPROVED** [review]
now that the other PR is merged, I approve of a new release as well

**Review by saibotk — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -24,16 +26,17 @@ All notable changes to TTT2 will be documented here. Inspired by [keep a changel
   - Added portuguese translation
   - Added a `database` library, that handles shared Interaction with the sql database
```
Comment (saibotk) [inline CHANGELOG.md:27]: would you mind correcting the indentation on these lines here?

**Review by saibotk — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (ZenBre4ker) [reply CHANGELOG.md:27]: yeah sure, I corrected them once, but somehow in alll those updates I chose the wrong version

**Review by saibotk — APPROVED** [review]

**Pull request merged**